recommender/recommender.py

```diff
@@ -968,6 +970,23 @@ def studio_view(self, _context=None):  # pylint: disable=unused-argument
         frag.initialize_js('RecommenderXBlock')
         return frag
 
+    def add_xml_to_node(self, node):
+        """
+        Serialize the XBlock to XML for exporting.
+        """
+        node.tag = 'recommender'
+
+        node.set('intro_enabled', str(self.intro_enabled))
+        node.set('DISABLE_DEV_UX', str(self.client_side_settings['DISABLE_DEV_UX']))
+        node.set('ENTRIES_PER_PAGE', str(self.client_side_settings['ENTRIES_PER_PAGE']))
+        node.set('PAGE_SPAN', str(self.client_side_settings['PAGE_SPAN']))
+
+        el = etree.SubElement(node, 'resources')
+        ## Note: The line below does not work in edX platform. 
+        ## We should figure out if the appropriate scope is available during import/export
+        ## TODO: Talk to Cale
+        el.text = unicode(json.dumps(self.recommendations))
```

Comment:
Why are you casting to `unicode`? If you want to convert `str` to `unicode`, the correct way to do it is via the `decode` method, so you specify the encoding:
```python
el.text = json.dumps(self.recommendations).decode("utf-8")
```